include/host-build.mk

```diff
@@ -24,6 +24,7 @@ BUILD_TYPES += host
 HOST_STAMP_PREPARED=$(HOST_BUILD_DIR)/.prepared$(if $(HOST_QUILT)$(DUMP),,$(shell $(call find_md5,${CURDIR} $(PKG_FILE_DEPENDS),))_$(call confvar,CONFIG_AUTOREMOVE $(HOST_PREPARED_DEPENDS)))
 HOST_STAMP_CONFIGURED:=$(HOST_BUILD_DIR)/.configured
 HOST_STAMP_BUILT:=$(HOST_BUILD_DIR)/.built
+HOST_STAMP_REMOVED:=$(HOST_BUILD_DIR)/.removed
```

Comment:
For package.mk this was added to call clean-build. For host-build.mk what is the use of such file? Consistency?

Reply:
consistency, and who knows maybe actually used for something in the future

Reply:
also IMHO we should keep the autoremove naming just it's a nitpick